addons/amb_civ_population/fnc_civInteract.sqf

```diff
@@ -783,6 +798,39 @@ switch (_operation) do {
 		};
 	};
 
+	case "giveItem": {
+		_arguments params ["_itemType"];
+
+		_civ = [_logic, "Civ"] call ALiVE_fnc_hashGet;
+		_humanitarian = [_logic, "humanitarianData"] call ALiVE_fnc_hashGet;
+		_item = [_humanitarian, _itemType] call ALiVE_fnc_hashGet;
+		_decreaseChance = [_humanitarian, "hostilityDecrease"] call ALiVE_fnc_hashGet;
+		_maxAllowAid = [_humanitarian, "maxAllowAid"] call ALiVE_fnc_hashGet;
+
+		// Check amount of aid already received
+		_consumed = _civ getVariable[QGVAR(consumedItems), 0];
+		if (_consumed >= (parseNumber _maxAllowAid)) exitWith {
```

Comment:
Not really a big deal, but typically if a value coming off the module is a string but it's supposed to be a number (_maxAllowAid), then we handle that during conversion during module init so we don't have to convert each time we use it.